wallet-sidebar.js

```diff
@@ -21,12 +21,7 @@ const sidebar = {
       type: "category",
       label: "Tutorials",
       link: { type: "generated-index", slug: "/tutorials" },
-      items: [
-        {
-          type: "doc",
-          id: "tutorials/react-dapp-local-state",
-        },
-      ],
+      items: [{ type: "autogenerated", dirName: "concepts" }]
```

Comment:
This was updated in https://github.com/MetaMask/metamask-docs/pull/1299 because we wanted to remove the global tutorial from the sidebar but keep it as existing page. So we need to revert this back to manual list and add the new tutorial here.